view/frontend/web/js/action/dintero-payment-service.js

```diff
@@ -11,7 +11,7 @@ define(
         'use strict';
         return {
             currentRequest: false,
-            init: function() {
+            init: async function() {
```

Comment:
using async/await will probably fail for older devices (pre 2017) without updates, I think it's better to initialize currentRequest inside the `then`, like this:

```js
storage.post(serviceUrl, JSON.stringify(payload), true, 'application/json')
    .then(function(session) {
        this.currentRequest = dintero.embed({
            container: $('#dintero-embedded-checkout-container').get(0),
            sid: session.id,
            language: window.checkoutConfig.payment.dintero.language,
            onPaymentError: function(event, checkout) {
                $(_this).trigger(
                    'dintero.payment.failed',
                    $.mage.__('The payment was out of date. Refresh the page to try again')
                );
                checkout.destroy();
            }
        });
```

Reply:
@andersem , I've added similar changes, with the exception `this.currentRequest `should be assigned ajax request object.

Reply:
> @andersem , I've added similar changes, with the exception `this.currentRequest `should be assigned ajax request object.

Aha, I misunderstood. Thanks.